packages/expo/src/migrations/update-16-6-0/update-metro-config.ts

```diff
@@ -0,0 +1,58 @@
+import { Tree, getProjects } from '@nx/devkit';
+import { join } from 'path';
+
+/**
+ * This migration updates metro.config.js to export config as a default.
+ *
+ */
+export default async function update(tree: Tree) {
+  const projects = getProjects(tree);
+
+  for (const [_, config] of projects.entries()) {
+    if (config.targets?.['start']?.executor === '@nx/expo:start') {
+      if (tree.exists(join(config.root, 'metro.config.js'))) {
```

Comment:
What happens with their existing configuration? Should we maybe rename the old configuration and add a comment for them to port any customizations over?

Reply:
i change this migration script. i will rename the old metro.config to metro-v71.config. i added a comment on top of the old metro config.